### DISCUSSION TITLE
GitHub Private Repositories CI/CD Integration Deployments getting cancelled

### DISCUSSION THREAD

**Discussion by jay-babu** [serverless/serverless#12291] [Q&A] [answered]
I started using Serverless today and have been having some trouble setting up CI/CD. I researched on github and serverless forums a lot before asking this. I integrated serverless to do a CI/CD deployment in a GitHub private repo as configured here:

![image](https://github.com/serverless/serverless/assets/36803168/3bdf6ee7-9f8f-4f19-98c2-33875d782dfc)

The deployment logs are just showing cancelled in the preflight information with no other logs.

<img width="1280" alt="image" src="https://github.com/serverless/serverless/assets/36803168/79f09475-d821-4eeb-91e5-ed4ea400e96b">

![image](https://github.com/serverless/serverless/assets/36803168/792f8f54-09f8-4b6f-b1c4-89d25ebeb93a)

Unsure how to go about this with no other logged information. I did not hit cancel or anything. running `sls deploy` from the cli works fine

**Comment by scottmurray14** [comment]
@jay-babu did you ever get to the bottom of this?

Happening here too..

**Reply by jay-babu** [reply]
Nope

**Reply by scottmurray14** [reply, marked as answer]
@jay-babu just spoke to their support.

Turns out our free account doesn't work with GitHub enterprise accounts/repositories, so they suggested we upgrade our serverless account.

Not sure if that applies to you but FYI.

**Reply by jay-babu** [reply]
yes that is it. Thanks!
